package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless

```diff
@@ -226,6 +227,8 @@ wireless.foreach_radio(uci, function(radio, index, config)
 		end
 	end
 
+	uci:set('wireless', radio_name, 'beacon_int', beacon_interval)
```

Comment:
what happens here if beacon_interval is nil because it's not defined in the site?

Reply:
ok, it seems if "nil" is supplied, uci:set does simply delete the setting from the config.